### PR TITLE
fix: don't directly throttle unsupported/privileged transactions

### LINKED ISSUE #12546: Refine throttle checks at ingest
### Description

A non-privileged `0.0.X` payer gets `BUSY` when submitting a privileged (e.g. `Freeze`) or unsupported (e.g. `AddLiveHash`) transaction.

It would make more sense to give immediate failure statuses of `UNAUTHORIZED` and `NOT_SUPPORTED`, respectively.

### Steps to reproduce

As above.

### Additional context

_No response_

### Hedera network

previewnet

### Version

`develop`

### Operating system

None

### PR DESCRIPTION
**Pull request by tinker-michaelj**
**Description**:
 - Closes #12546 
 - Since unsupported and privileged transactions are not assigned a throttle bucket, screen them differently at ingest.
     * For an unsupported transaction, fail precheck with `NOT_SUPPORTED`.
     * For a privileged transaction submitted by a user account, fail precheck with `UNAUTHORIZED`.

### CODE REVIEW

**Review by Neeharika-Sompalli — APPROVED** [review]
LGTM ! Thanks @tinker-michaelj .

File: hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/ingest/IngestChecker.java [hunk 1]
```diff
@@ -242,6 +255,26 @@ public TransactionInfo runAllChecks(
...
+            throws PreCheckException {
+        if (UNSUPPORTED_TRANSACTIONS.contains(txInfo.functionality())) {
+            throw new PreCheckException(NOT_SUPPORTED);
+        }
+        if (PRIVILEGED_TRANSACTIONS.contains(txInfo.functionality())) {
+            final var payerNum =
+                    txInfo.payerID() == null ? Long.MAX_VALUE : txInfo.payerID().accountNumOrElse(Long.MAX_VALUE);
+            final var hederaConfig = configuration.getConfigData(HederaConfig.class);
+            // This adds a mild restriction that privileged transactions can only
+            // be issued by system accounts; (FUTURE) consider giving non-trivial
```
Comment (Neeharika-Sompalli) [inline hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/ingest/IngestChecker.java:269]: Good to have a  FUTURE to re-look again 💯

**Review by povolev15 — APPROVED** [review]
LGTM

File: hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/ingest/IngestChecker.java (hunk 1 above)
Comment (tinker-michaelj) [reply hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/ingest/IngestChecker.java:277]: Added link to issues as well. 💯

**Pull request merged**